token/js/test/e2e-2022/Amount.test.ts

```diff
@@ -0,0 +1,45 @@
+import chai, { expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+chai.use(chaiAsPromised);
+
+import { Connection, Keypair, PublicKey, Signer } from '@solana/web3.js';
+
+import { createMint, amountToUiAmount, uiAmountToAmount } from '../../src';
+
+import { TEST_PROGRAM_ID, newAccountWithLamports, getConnection } from '../common';
+
+const TEST_TOKEN_DECIMALS = 2;
+describe('Amount', () => {
+    let connection: Connection;
+    let payer: Signer;
+    let mint: PublicKey;
+    let mintAuthority: Keypair;
+    let account: PublicKey;
+    before(async () => {
+        connection = await getConnection();
+        payer = await newAccountWithLamports(connection, 1000000000);
+        mintAuthority = Keypair.generate();
+        const mintKeypair = Keypair.generate();
+        mint = await createMint(
+            connection,
+            payer,
+            mintAuthority.publicKey,
+            mintAuthority.publicKey,
+            TEST_TOKEN_DECIMALS,
+            mintKeypair,
+            undefined,
+            TEST_PROGRAM_ID
+        );
+    });
+    it('amountToUiAmount', async () => {
+        const amount = BigInt(5245);
+        const uiAmount = await amountToUiAmount(connection, payer, mint, amount, TEST_PROGRAM_ID);
+        expect(uiAmount).to.be.not.null;
```

Comment:
nit: It's not necessary to test not null since you're testing its equality right after